test/hdimage-fail3.config

```diff
@@ -0,0 +1,9 @@
+image test.hdimage {
+	hdimage {
+		partition-table-type = "hybrid"
+	}
+	partition part1 {
+		/* missing partition-type */
+		partition-type-uuid = "L"
```

Comment:
We should probably also make sure we are testing hybrid partitions similar to how they would be used in practice. In particular we should check that the new equivalent of this boot entry for my [raspberry pi config](https://github.com/pengutronix/genimage/pull/96#issue-392132105) works.
```
  partition boot {
    partition-type = 0xC
    partition-type-uuid = c12a7328-f81f-11d2-ba4b-00a0c93ec93b
    bootable = "true"
    image = "boot.vfat"
  }
```
It might also be a good idea to have tests for the following:

- Ensure that more than 3 MBR partitions in a hybrid image fails
- Ensure that a hybrid image with a mixture of hybrid partition entries and GPT only partition entries succeeds. This is the primary way I use hybrid partitions as I only need the MBR partition boot entry so that the raspberry pi can read the firmware and kernel from the boot partition, the kernel itself once launched will operate as if the disk is a normal GPT disk at this point.